kuberay/manifests/ray-cluster.tpu-v4-multihost.yaml

```diff
@@ -0,0 +1,144 @@
+# This template contains a Kuberay cluster using a 2x2x2 TPU v4 PodSlice.
+# To get access to TPU resources, please follow instructions in this link:
+# https://cloud.google.com/kubernetes-engine/docs/how-to/tpus
+apiVersion: ray.io/v1
+kind: RayCluster
+metadata:
+  name: example-cluster-kuberay
+spec:
+  headGroupSpec:
+    rayStartParams:
+      {}
+    template:
+      spec:
+        imagePullSecrets:
+          []
+        serviceAccountName: ray-ksa
+        containers:
+          - volumeMounts:
+            - name: gcs-fuse-checkpoint
+              mountPath: /llama
+              readOnly: true
+            - mountPath: /tmp/ray
+              name: ray-logs
+            name: ray-head
+            image: gcr.io/tpu-vm-gke-testing/ricliu-jetstream:20240709
+            imagePullPolicy: IfNotPresent
+            resources:
+              limits:
+                cpu: "4"
+                ephemeral-storage: 30Gi
+                memory: 40G
+              requests:
+                cpu: "4"
+                ephemeral-storage: 30Gi
+                memory: 40G
+            securityContext:
+              {}
+            env:
+              - name: JAX_PLATFORMS
+                value: "cpu"
+              - name: RAY_memory_monitor_refresh_ms
+                value: "0"
+              - name: RAY_GRAFANA_IFRAME_HOST
+                value: http://${grafana_host}
+              - name: RAY_GRAFANA_HOST
+                value: http://grafana:80
+              - name: RAY_PROMETHEUS_HOST
+                value: http://frontend:9090
+            ports:
+              - containerPort: 6379
+                name: gcs
+              - containerPort: 8265
+                name: dashboard
+              - containerPort: 10001
+                name: client
+              - containerPort: 8000
+                name: serve
+              - containerPort: 8471
+                name: slicebuilder
+              - containerPort: 8081
+                name: mxla
+              - containerPort: 8888
+                name: grpc
+        volumes:
+          - emptyDir: {}
+            name: ray-logs
+          - name: gcs-fuse-checkpoint
+            csi:
+              driver: gcsfuse.csi.storage.gke.io
+              readOnly: true
+              volumeAttributes:
+                bucketName: ricliu-llama2-70b-chat
```

Comment:
This bucket is not publically available, can we host it in a public bucket or provide instructions to push the model weights to a bucket?

Reply:
cc @ryanaoleary